web/src/LoadingScreen.scss

```diff
@@ -1,8 +1,14 @@
+@import "constants";
+
 .LoadingScreen {
-  min-height: 100vh;
+  position: fixed;
```

Comment:
this seems wrong to me? now the loading screen is off-center and looks kind of broken